lib/active_model/serializer/configuration.rb

```diff
@@ -26,6 +26,7 @@ def config.array_serializer
         # Make JSON API top-level jsonapi member opt-in
         # ref: http://jsonapi.org/format/#document-top-level
         config.jsonapi_include_toplevel_object = false
+        config.key_transform = nil
```

Comment:
oh n/m, that let's you configure per adapter. but this raise a point..

if this value is set to anything truthy, it will affect all adapters.  Is that the intention?